app/views/evaluation_forms/_evaluation_criterion_fields.html.erb

```diff
@@ -0,0 +1,192 @@
+    aria-controls="<%= criteria_field_id(f, "accordion", is_template) %>"
+    data-evaluation-criteria-target="accordionButton"
+    data-action="click->evaluation-criteria#validateInputs"
+  >
+    <div>Criteria <span class="criteria-number" data-evaluation-criteria-target="criteriaNumber"><%= f.options[:child_index] + 1 %></span></div>
+  </button>
+</h4>
+
+<div id="<%= criteria_field_id(f, "accordion", is_template) %>" class="usa-accordion__content" data-evaluation-criteria-target="accordionContent">
+  <% if f.object.persisted? %>
+    <%= f.hidden_field :id, value: f.object.id %>
+    <%= f.hidden_field :_destroy, class: "destroy-evaluation-criteria", data: {"evaluation-criteria-target": "destroyField"} %>
+  <% end %>
+
+  <div class="usa-form-group margin-top-0">
+    <%= f.label :title, "Criteria title", for: criteria_field_id(f, "title", is_template), class: "text-bold" %>
+    <span style="color: darkred;">*</span>
+    <%= image_tag "images/usa-icons/help_outline.svg", width: 16, height: 16, alt: "Help for criteria title" %>
+
+    <%= f.text_field :title, 
+      id: criteria_field_id(f, "title", is_template),
+      name: criteria_field_name(f, "title", is_template),
+      class: "usa-input", 
+      placeholder: "Add criteria title here", 
+      maxlength: 150,
+      required: true, 
+      disabled: is_template || form_disabled,
+      data: {"evaluation-criteria-target": "titleField"}
+    %>
+  </div>
+
+  <div class="usa-character-count">
+    <div class="usa-form-group">
+      <%= f.label :description, "Criteria description", for: criteria_field_id(f, "description", is_template), class: "text-bold" %>
+      <span style="color: darkred;">*</span>
+      <%= image_tag "images/usa-icons/help_outline.svg", width: 16, height: 16, alt: "Help for criteria title" %>
+
+      <%= f.text_area :description, 
+        id: criteria_field_id(f, "description", is_template),
+        name: criteria_field_name(f, "description", is_template),
+        class: "usa-textarea usa-character-count__field", 
+        placeholder: "Add criteria description here", 
+        maxlength: 1000,
+        required: true, 
+        disabled: is_template || form_disabled,
+        data: {"evaluation-criteria-target": "descriptionField"}
+      %>
+    </div>
+    <span class="usa-character-count__message">You can enter up to 1000 characters</span>
+  </div>
+
+  <div class="usa-form-group display-flex flex-align-center">
+    <%= f.number_field :points_or_weight, 
+      id: criteria_field_id(f, "points_or_weight", is_template),
+      name: criteria_field_name(f, "points_or_weight", is_template),
+      class: "usa-input flex-1 margin-top-0 margin-right-2 font-sans-lg", 
+      min: 1, 
+      step: 1, 
+      placeholder: "Add criteria points/weight here",
+      required: true, 
+      disabled: is_template || form_disabled,
+      data: {"evaluation-criteria-target": "pointsOrWeightField"}
```

Comment:
it looks like some of these data -target attributes are not being used, like this one?

Reply:
Yeah I'll go through and clean these. This was when I was initially thinking of approaching the Stimulus thing a different way. Which might still work but I didn't try it because I was skeptical. The multiple of the same controller scoped to its own targets approach for the criteria and then one controller for the list itself.